lib/OpenQA/Schema/Schema.pm

```diff
@@ -24,12 +25,22 @@ use FindBin qw($Bin);
 # after bumping the version please look at the instructions in the docs/Contributing.asciidoc file
 # on what scripts should be run and how
 our $VERSION = 24;
+our $configname="$Bin/../lib/database.ini";
 
 __PACKAGE__->load_namespaces;
 
-my @paths = ( "$Bin/../lib/database", "$Bin/../../lib/database" );
-unshift(@paths, dirname($ENV{OPENQA_CONFIG}).'/database') if ($ENV{OPENQA_CONFIG});
-__PACKAGE__->config_paths(\@paths);
+
+sub connect_db {
+    my $mode = shift || $ENV{OPENQA_DATABASE} || 'production';
+    CORE::state $schema;
+    unless ($schema) {
+        my %ini;
+        #        print("ENV{OPENQA_DATABASE_CONFIG}=$ENV{OPENQA_DATABASE_CONFIG}\n");
+        tie %ini, 'Config::IniFiles', ( -file => $ENV{OPENQA_DATABASE_CONFIG} || $configname );
+        $schema=__PACKAGE__->connect($ini{$mode});
```

Comment:
oh, I see, sorry, did not noted your change before pull request. I did change it because from the code I had an impression that OPENQA_CONFIG is only for openqa.ini. It was also originaly done because of the test scripts, which, if in deeper level, could not find the database.ini file ../lib (../../lib was needed). Since in the OpenQA::Test::Case was already a OPENQA_CLIENT_CONFIG variable, it seemed natural to have another independent path for database.ini, in something like OPENQA_DATABASE_CONFIG.

If you think it is better as it was described in the documentation, I can change it to derive from OPENQA_CONFIG, but I still think, since we have already independent OPENQA_CLIENT_CONFIG, the database.ini may be justified to have its own env variable.